jetstream/metadata.py

```diff
@@ -25,6 +25,7 @@ class OutcomeMetadata:
     friendly_name: str
     description: str
     metrics: List[str]
+    commit_hash: Optional[str]
```

Comment:
This is the outcomes metadata currently included in the `metadata.json` that is published. Am I missing something else that we want to include here?